europeana-entity-harvester/index.html

```diff
@@ -0,0 +1,189 @@
+            .then(entity => {
+              if (entity.valid) {
+                populateFields(entity.json);
+                sendMessage('Success');
+              } else {
+                showError(entity.error);
+              }
+            });
+        }
+
+        async function entityFromApi(entityId) {
+          const apiKey = extension.parameters.installation.apiKey;
+          const response = await fetch(`https://api.europeana.eu/entity${entityId}.jsonld?wskey=${apiKey}`);
+          const responseJson = await response.json();
+
+          if (response.ok) {
+            return { valid: true, json: responseJson };
+          } else {
+            return { valid: false, error: responseJson.error ? responseJson.error : 'unknown' };
+          }
+        }
+
+        function entityDescriptionFromResponse(response) {
+          switch (entityTypeFromUri(response.id)) {
+            case 'agent':
+              // use `biographicalInformation`
+              // NB: this is in JSON-LD expanded form
+              return languageValue(response.biographicalInformation);
+              break;
+            case 'concept':
+              // use `note`
+              // NB: language map with each value being an array of literals
+              return languageValue(response.note);
+              break;
+            case 'organization':
+              // use `description`
+              // NB: language map with each value being a single literal
+              return languageValue(response.description);
+              break;
+            case 'place':
+              // TODO: use what? `${lat},${long}?`
+              return '';
+              break;
+          }
+        }
+
+        // TODO: set up a configurable map for other fields to avoid hard-coding them here
+        function populateFields(response) {
+          // set field values
+          extension.entry.fields.identifier.setValue(response.id); // data.europeana.eu URI
+
+          // set name field from `prefLabel`
+          if (extension.entry.fields.name) {
+            extension.entry.fields.name.setValue('');
+            if (response.prefLabel) {
+              extension.entry.fields.name.setValue(languageValue(response.prefLabel));
+            }
+          }
+
+          // set description from different fields based on entity type
+          if (extension.entry.fields.description) {
+            extension.entry.fields.description.setValue('');
+            extension.entry.fields.description.setValue(entityDescriptionFromResponse(response));
+          }
+
+          // set image field from `depiction`
+          if (extension.entry.fields.image) {
+            extension.entry.fields.image.setValue('');
+            if (response.depiction) {
+              extension.entry.fields.image.setValue(response.depiction.id);
+            }
+          }
+        }
+
+        // button click handler
+        document.getElementById('entityHarvest').addEventListener('click', e => {
```

Comment:
e is not used, so you can use () as well: 
`document.getElementById('entityHarvest').addEventListener('click', () => {`